docs/deployment_checklist.md

```diff
@@ -0,0 +1,72 @@
+# Deploying Apply - Post-deployment checks
+
+After deploying a new version of the service we have a short list of
+tests to run through. The aim is to detect any obvious issues that have
+been caused by the new release.
+
+This checklist is meant as a supplement to the full [deployment guide](deployment.md)
+where you will find detailed instructions about how to deploy.
+
+It may be appropriate to carry out additional checks specific to the
+changes included in a deployment.
+
+We aim to work in pairs on deployments so that one developer runs the
+deployment itself and a second developer runs through this checklist.
+
+## 1. Staging checks
+
+When the staging deployment has finished and before the production
+deployment begins ask another developer to go through the following
+checks:
+
+1. Notify the rest of the team via Slack that staging is deployed and
+   that final pre-production checks can be made.
+2. Check the #twd_apply_tech channel in Slack for runtime errors from
+   Sentry.
+3. Create a new account on
+   [staging](https://staging.apply-for-teacher-training.education.gov.uk/candidate)
+   with a real email address and check that you receive an email with
+   the magic link and can use it to log in. Use your DfE digital email
+   address so that we can distinguish test users.  If needed you can
+   suffix your name with `+n` to create a unique email address.
+4. Check the Azure Dashboard for the staging
+   [s106t01-apply](https://portal.azure.com/#@9c7d9dd3-840c-4b3f-818e-552865082e16/dashboard/private/6fef53c1-fbe3-4bd3-aa3d-8575eebb2424) resource
```

Comment:
That URL gives @chubberlisk and me 404s. This link seems to work: https://portal.azure.com/#@platform.education.gov.uk/dashboard/arm/subscriptions/c426dea0-793c-4ab6-9dbc-b45dbdd9ef24/resourceGroups/s106t01-apply/providers/Microsoft.Portal/dashboards/s106t01-apply-dashboard